sdk/python/pulumi_kubernetes/_tables.py

```diff
@@ -0,0 +1,913 @@
+    "ipBlock": "ip_block",
+    "ipFamily": "ip_family",
+    "iscsiInterface": "iscsi_interface",
+    "jobTemplate": "job_template",
+    "jsonPath": "json_path",
+    "kernelVersion": "kernel_version",
+    "kubeProxyVersion": "kube_proxy_version",
+    "kubeletConfigKey": "kubelet_config_key",
+    "kubeletEndpoint": "kubelet_endpoint",
+    "kubeletVersion": "kubelet_version",
+    "labelSelector": "label_selector",
+    "labelSelectorPath": "label_selector_path",
+    "lastHeartbeatTime": "last_heartbeat_time",
+    "lastKnownGood": "last_known_good",
+    "lastObservedTime": "last_observed_time",
+    "lastProbeTime": "last_probe_time",
+    "lastScaleTime": "last_scale_time",
+    "lastScheduleTime": "last_schedule_time",
+    "lastState": "last_state",
+    "lastTimestamp": "last_timestamp",
+    "lastTransitionTime": "last_transition_time",
+    "lastUpdateTime": "last_update_time",
+    "leaseDurationSeconds": "lease_duration_seconds",
+    "leaseTransitions": "lease_transitions",
+    "limitResponse": "limit_response",
+    "listKind": "list_kind",
+    "livenessProbe": "liveness_probe",
+    "loadBalancer": "load_balancer",
+    "loadBalancerIP": "load_balancer_ip",
+    "loadBalancerSourceRanges": "load_balancer_source_ranges",
+    "machineID": "machine_id",
+    "managedFields": "managed_fields",
+    "manualSelector": "manual_selector",
+    "matchExpressions": "match_expressions",
+    "matchFields": "match_fields",
+    "matchLabelExpressions": "match_label_expressions",
+    "matchLabels": "match_labels",
+    "matchPolicy": "match_policy",
+    "matchingPrecedence": "matching_precedence",
+    "maxItems": "max_items",
+    "maxLength": "max_length",
+    "maxLimitRequestRatio": "max_limit_request_ratio",
+    "maxProperties": "max_properties",
+    "maxReplicas": "max_replicas",
+    "maxSkew": "max_skew",
+    "maxSurge": "max_surge",
+    "maxUnavailable": "max_unavailable",
+    "metricName": "metric_name",
+    "metricSelector": "metric_selector",
+    "minAvailable": "min_available",
+    "minItems": "min_items",
+    "minLength": "min_length",
+    "minProperties": "min_properties",
+    "minReadySeconds": "min_ready_seconds",
+    "minReplicas": "min_replicas",
+    "mountOptions": "mount_options",
+    "mountPath": "mount_path",
+    "mountPropagation": "mount_propagation",
+    "multipleOf": "multiple_of",
+    "namespaceSelector": "namespace_selector",
+    "nodeAffinity": "node_affinity",
+    "nodeID": "node_id",
+    "nodeInfo": "node_info",
+    "nodeName": "node_name",
+    "nodePort": "node_port",
+    "nodePublishSecretRef": "node_publish_secret_ref",
+    "nodeSelector": "node_selector",
+    "nodeSelectorTerms": "node_selector_terms",
+    "nodeStageSecretRef": "node_stage_secret_ref",
+    "nominatedNodeName": "nominated_node_name",
+    "nonResourceAttributes": "non_resource_attributes",
+    "nonResourceRules": "non_resource_rules",
+    "nonResourceURLs": "non_resource_ur_ls",
```

Comment:
This looks wrong

Reply:
It's the same value we've always emitted:

https://github.com/pulumi/pulumi-kubernetes/blob/c4b1ff2ed580565cd919cdfe7d57d45df4a4e142/sdk/python/pulumi_kubernetes/tables.py#L720

Same comment as above.